README.md

```diff
@@ -3,123 +3,56 @@
-
-**See [NEWS](https://github.com/jump-dev/JuMP.jl/tree/master/NEWS.md) for
-a list of the significant breaking changes in the JuMP 0.19 release.**
-
-**Latest Release**: 0.21.5 (`release-0.21` branch)
-  * [Documentation](https://jump.dev/JuMP.jl/v0.21.5/)
+JuMP is a domain-specific modeling language for [mathematical optimization](https://en.wikipedia.org/wiki/Mathematical_optimization)
+embedded in [Julia](https://julialang.org/). You can find out more about us by 
+visiting [jump.dev](https://jump.dev).
+
+**Latest Release**: 0.21.5 (`release-0.21` branch):
+  * Use this version if you are using JuMP to solve problems.
+  * Installation via the Julia package manager:
+    * `import Pkg; Pkg.add("JuMP")`
+  * Get help:
+    * Read the [Documentation](https://jump.dev/JuMP.jl/v0.21.5/)
+    * Ask a question on the [Community forum]
+    * Explore the [JuMPTutorials](https://github.com/jump-dev/JuMPTutorials.jl)
   * Testing status:
     * Github Actions: [![Build Status](https://github.com/jump-dev/JuMP.jl/workflows/CI/badge.svg?branch=release-0.21)](https://github.com/jump-dev/JuMP.jl/actions?query=workflow%3ACI)
 
-
 **Development version** (`master` branch):
-  * [Documentation](https://jump.dev/JuMP.jl/dev/)
+  * Use this version if you are hacking on JuMP or related projects.
+  * Installation via the Julia package manager:
+    * `import Pkg; Pkg.add(Pkg.PackageSpec(name="JuMP", rev="master"))`
+  * Get help:
+    * Read the [Documentation](https://jump.dev/JuMP.jl/dev/)
+    * Join the [Developer chatroom](https://gitter.im/JuliaOpt/JuMP-dev)
+    * Read the [NEWS](https://github.com/jump-dev/JuMP.jl/tree/master/NEWS.md)
   * Testing status:
     * Github Actions: [![Build Status](https://github.com/jump-dev/JuMP.jl/workflows/CI/badge.svg?branch=master)](https://github.com/jump-dev/JuMP.jl/actions?query=workflow%3ACI)
     * Test coverage: [![codecov](https://codecov.io/gh/jump-dev/JuMP.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/jump-dev/JuMP.jl)
-  * Changes: see [NEWS](https://github.com/jump-dev/JuMP.jl/tree/master/NEWS.md)
-  * [Developer chatroom](https://gitter.im/JuliaOpt/JuMP-dev)
-
-
-## Installation
-
-JuMP can be installed through the Julia package manager:
-
-```julia
-julia> Pkg.add("JuMP")
-```
-
-For full installation instructions, including how to install solvers, see the documentation linked above.
 
+## Need help?
 
-## Supported problem classes
+Use the [Community forum] to search for answers to previously asked questions, 
+or ask a new question. 
 
-Mathematical optimization encompasses a large variety of problem classes.
-We list below what is currently supported. See the documentation for more information.
+Before asking a question, you can help us by reading [PSA: make it easier to help you](https://discourse.julialang.org/t/psa-make-it-easier-to-help-you/14757),
+and following the suggestions. In particular, _simplify_ your problem and make
```

Comment:
Could drop "In particular, _simplify_ your problem and make it reproducible". The sentence could be interpreted as somewhat condescending. I'd prefer to phrase this more like: "here are the best practices for asking a question."